mne/decoding/time_gen.py

```diff
@@ -356,6 +356,7 @@ def predict(self, epochs):
         if self.test_times == 'diagonal':
             test_times = _DecodingTime()
             test_times['slices'] = [[s] for s in self.train_times_['slices']]
+            test_times['times'] = [[s] for s in self.train_times_['times']]
```

Comment:
It's a list of arrays, because the number of testing times per training times is not necessarily fixed. 

I could do `self.train_times_['times'].copy()[:, np.newaxis].tolist()` if you want?